liota/core/package_manager.py

```diff
@@ -289,7 +287,19 @@ def _cmd_handler_list(self, parameter):
     # This method is used to handle statistical commands
 
     def _cmd_handler_stat(self, parameter):
-        if parameter == "metrics" or parameter == "met":
+        if parameter[0] == "package" or parameter[0] == "pkg":
```

Comment:
before call _cmd_handler_stat(), we have checked and guaranteed there will be at least 1 parameter, which can be "met, pkg, etc..". So, we can always use parameter[0] rather than parameter to get "pkg". After this, I have sanity check to guarantee that after "pkg", we have more package_names as parameters to process.

Reply:
I meant, shouldn't the name of the variable be "parameters" instead of parameter? It seems incorrect to call it parameter, since it is multiple.

Reply:
in fact, so far, only "stat pkg" has further parameter, "stat met/th/col" will have no further parameter. But because they will share the same _cmd_handler_stat(), I have to use parameter[0] to process for all of them.
Yes, i agree even in above case, parameters will be more suitable as parameter. Will change it.

Reply:
done